modules/Core/classes/Events/UserIntegrationLinkedEvent.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+class UserIntegrationLinkedEvent extends AbstractEvent implements DiscordDispatchable {
```

Comment:
Well most event class names sounds like the event is complete, adding the ability to setCancelled later will make the event file names sounds weird So i guess the event names should had stayed as they were

Example this is linkIntegrationUser with can still be cancellable but the name UserIntegrationLinkedEvent does not sound like that